pkg/services/store/service.go

```diff
@@ -86,3 +108,82 @@ func (s *standardStorageService) Read(ctx context.Context, user *models.SignedIn
 	// TODO: permission check!
 	return s.tree.GetFile(ctx, path)
 }
+
+func isFileTypeValid(filetype string) bool {
+	if (filetype == "image/jpeg") || (filetype == "image/jpg") || (filetype == "image/gif") || (filetype == "image/png") || (filetype == "image/svg+xml") || (filetype == "image/webp") {
```

Comment:
Could thee filetypes be configurable somehow? Or are these the only types we can process?

Reply:
soon -- this should be configured in the store, but for now hardcoded is OK.  In particular, we will need the check to be much more robust.  ie: we need to make sure an uploaded image is actually an image and not an executable!

But for starters, this feels like progress :)